src/ui/EditTask.svelte

```diff
@@ -273,7 +275,7 @@
 <div class="tasks-modal">
     <form on:submit|preventDefault={_onSubmit}>
         <div class="tasks-modal-section">
-            <label for="description">Descrip<span class="accesskey">t</span>ion</label>
+            <label for="description">Descrip<span class:accesskey="{withAccessKeys}">t</span>ion</label>
```

Comment:
For my own education, how does this bit of the change work please?

Reply:
It's a Svelte template language idiom to conditionally set a CSS class (in this case the class "accesskey").

I have changed this now a bit and always statically set the "accesskey" class on the elements, and make only an additional top level class "with-accesskeys" dynamic.